src/BusinessLayer/DataPopulators/JsonDefines.h

```diff
@@ -221,9 +215,23 @@ namespace JsonFormat
     const QString AUXBMS_AUXBMSALIVE = "AuxBmsAlive";
     const QString AUXBMS_STROBEBMSLIGHT = "StrobeBmsLight";
     const QString AUXBMS_ALLOWCHARGE = "AllowCharge";
-    const QString AUXBMS_CONTACTORERROR = "ContactorError";
-    const QString AUXBMS_HIGHVOLTAGEENABLE = "HighVoltageEnable";
 
+    const QString AUXBMS_CHARGETRIP_HIGHVOLT = "ChargeTripDueToHighCellVoltage";
```

Comment:
@bill-luu What do you think about abbreviating some of these?

Reply:
What kind of abbreviations did you have in mind?

Reply:
And do you mean abbreviations when displaying, or in the code?

Reply:
I mean using `AUXBMS_CHARGETRIP_HIGHVOLT ` instead of something like `CHARGE_TRIP_DUE_TO_HIGH_CELL_VOLTAGE` in the code

Reply:
Sounds good to me!